jobs/garbage-collection.Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             choices: pipeutils.get_streams_choices(pipecfg),
+             description: 'CoreOS stream to run GC')
+    ])
+])
+
+def build_description = "[${params.STREAM}] Garbage Collection"
+
+try {
+    currentBuild.description = "${build_description} Running"
+
+    credentials = [file(variable: "GCP_IMAGE_UPLOAD_CONFIG",
+                    credentialsId: "gcp-image-upload-config")]
+    // pick up the GCP project to use from the config
+    def gcp_project = shwrapCapture("jq -r .project_id \${GCP_KOLA_TESTS_CONFIG}")
+    def acl = pipecfg.s3.acl ?: 'public-read'
+
+    if (utils.credentialsExist(credentials)){
+        stage('Garbage Collection') {
+            pipeutils.shwrapWithAWSBuildUploadCredentials("""
+            cosa cloud-prune --policy ../gc-policy.yaml \
+            --stream ${params.STREAM} \
+            --dry-run \
```

Comment:
ADding `dry-run` for now till we get the initial run and we can make everything looks good. Can also add it as a parameter to the run but we would need to pass that through from the release job itself. Wasn't sure about that.

Reply:
Yep, actually did end up adding the parameter for that reason.

Reply:
yeah cool I just noticed that doing a review :)